README.md

```diff
@@ -81,13 +77,12 @@ and the passwordFile is used for encrypting/decrypting the private key. The othe
 
 #### Generate Keystore
 
-Before generate keystore, you should export your private key from MetaMask and write it into a local file as plaintext .
-You need also write your password on the password file which set by the "passwordFile" field in the config file.
+Before generate keystore, you should export your private key from MetaMask and write it into a local file as plaintext.
 
 Assuming that the current private key hex string is written as plaintext in the file key.txt, the following command can be used to generate a keystore file called key.json:
 ```
 // generate keystore key.json
-gnfd-cmd create-keystore --privKeyFile key.txt key.json
+gnfd-cmd create-keystore --privKeyFile key.txt 
 ```
 
 After the keystore file has been generated, you can delete the private key file which contains the plaintext of private key.
```

Comment:
Tell user where the generated keystore file is saved, and also the password file.

Reply:
updated